relnotes/v0.82.0.md

```diff
@@ -0,0 +1,9 @@
+### New features
+
+* [#7867](https://github.com/rubocop-hq/rubocop/issues/7867): Add support for tabs in indentation. ([@DracoAter][])
```

Comment:
This file is auto-generated on release from the changelog. You shouldn't update it manually.